sdk/core/azure-core/test/ut/transport_policy_options.cpp

```diff
@@ -448,12 +452,13 @@ namespace Azure { namespace Core { namespace Test {
         auto request = Azure::Core::Http::Request(Azure::Core::Http::HttpMethod::Get, testUrl);
         auto response = pipeline.Send(request, Azure::Core::Context::ApplicationContext);
         EXPECT_EQ(response->GetStatusCode(), Azure::Core::Http::HttpStatusCode::Ok);
+        break; // Test passed, no need to retry.
       }
-      catch (Azure::Core::Http::TransportException const&)
+      catch (Azure::Core::Http::TransportException const& ex)
       {
         // CURL returns a connection error which triggers a transport exception.
```

Comment:
Is this really a curl-only error? It seems like this is all transports.

Reply:
Will remove. There's another hit of this in the ifdef'd test `ProxyWithPasswordHttps`. Shall I remove the comment there too?

Reply:
Hmm. I'm torn here. I'd like everything to be the same, but I'd rather not defocus this PR. It's ultimately your call.